Makefile

```diff
@@ -0,0 +1,63 @@
+NAME=cimgui-go Code Generator
+GOCMD=LC_ALL=C go
+TIMEOUT=5
+
+# go tools
+export PATH := ./bin:$(PATH)
+export GO111MODULE := on
+export GOPROXY = https://proxy.golang.org,direct
+
+.PHONY: all setup cimgui cimplot generate
```

Comment:
Let's do it like this:

```make
.PHONY: all
all:
        ...
```

for all targets. Putting `PHONY` in a separate place is confusing.